--- libpng-1.6.2/libpng-config.in.0003~	2013-04-25 14:24:44.000000000 +0200
+++ libpng-1.6.2/libpng-config.in	2013-06-15 09:57:52.532942702 +0200
@@ -14,12 +14,12 @@
 version="@PNGLIB_VERSION@"
 prefix="@prefix@"
 exec_prefix="@exec_prefix@"
-libdir="@libdir@"
+libdir=""
 includedir="@includedir@/libpng@PNGLIB_MAJOR@@PNGLIB_MINOR@"
 libs="-lpng@PNGLIB_MAJOR@@PNGLIB_MINOR@"
 all_libs="-lpng@PNGLIB_MAJOR@@PNGLIB_MINOR@ @LIBS@"
 I_opts="-I${includedir}"
-L_opts="-L${libdir}"
+L_opts=""
 R_opts=""
 cppflags=""
 ccopts=""
--- libpng-1.6.2/libpng.pc.in.0003~	2013-04-25 14:24:44.000000000 +0200
+++ libpng-1.6.2/libpng.pc.in	2013-06-15 09:57:52.532942702 +0200
@@ -1,11 +1,11 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-libdir=@libdir@
+libdir=
 includedir=@includedir@/libpng@PNGLIB_MAJOR@@PNGLIB_MINOR@
 
 Name: libpng
 Description: Loads and saves PNG files
 Version: @PNGLIB_VERSION@
-Libs: -L${libdir} -lpng@PNGLIB_MAJOR@@PNGLIB_MINOR@
+Libs: -lpng@PNGLIB_MAJOR@@PNGLIB_MINOR@
 Libs.private: @LIBS@
 Cflags: -I${includedir}
--- libpng-1.6.2/scripts/libpng.pc.in.0003~	2013-06-15 09:57:52.532942702 +0200
+++ libpng-1.6.2/scripts/libpng.pc.in	2013-06-15 09:58:30.118472818 +0200
@@ -1,10 +1,9 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-libdir=@libdir@
 includedir=@includedir@/libpng16
 
 Name: libpng
 Description: Loads and saves PNG files
 Version: 1.6.2
-Libs: -L${libdir} -lpng16
+Libs: -lpng16
 Cflags: -I${includedir}
