--- libpng-1.6.22/libpng-config.in.0001~	2016-05-26 05:41:46.000000000 +0200
+++ libpng-1.6.22/libpng-config.in	2016-06-07 15:23:20.393453633 +0200
@@ -14,12 +14,12 @@
 version="@PNGLIB_VERSION@"
 prefix="@prefix@"
 exec_prefix="@exec_prefix@"
-libdir="@libdir@"
+libdir=
 includedir="@includedir@/libpng@PNGLIB_MAJOR@@PNGLIB_MINOR@"
 libs="-lpng@PNGLIB_MAJOR@@PNGLIB_MINOR@"
 all_libs="-lpng@PNGLIB_MAJOR@@PNGLIB_MINOR@ @LIBS@"
 I_opts="-I${includedir}"
-L_opts="-L${libdir}"
+L_opts=
 R_opts=""
 cppflags=""
 ccopts=""
--- libpng-1.6.22/libpng.pc.in.0001~	2016-05-26 05:41:46.000000000 +0200
+++ libpng-1.6.22/libpng.pc.in	2016-06-07 15:23:20.393453633 +0200
@@ -1,11 +1,11 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-libdir=@libdir@
+libdir=
 includedir=@includedir@/libpng@PNGLIB_MAJOR@@PNGLIB_MINOR@
 
 Name: libpng
 Description: Loads and saves PNG files
 Version: @PNGLIB_VERSION@
-Libs: -L${libdir} -lpng@PNGLIB_MAJOR@@PNGLIB_MINOR@
+Libs: -lpng@PNGLIB_MAJOR@@PNGLIB_MINOR@
 Libs.private: @LIBS@
 Cflags: -I${includedir}
--- libpng-1.6.22/scripts/libpng.pc.in.0001~	2016-06-07 15:23:20.394453630 +0200
+++ libpng-1.6.22/scripts/libpng.pc.in	2016-06-07 15:24:41.353235784 +0200
@@ -1,10 +1,10 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-libdir=@libdir@
+libdir=
 includedir=@includedir@/libpng16
 
 Name: libpng
 Description: Loads and saves PNG files
 Version: 1.6.22
-Libs: -L${libdir} -lpng16
+Libs: -lpng16
 Cflags: -I${includedir}
