diff -Naur libpng-1.6.17/libpng-config.in libpng-1.6.17.tpg/libpng-config.in
--- libpng-1.6.17/libpng-config.in	2015-03-26 13:06:23.000000000 +0000
+++ libpng-1.6.17.tpg/libpng-config.in	2015-04-08 18:12:11.560274002 +0000
@@ -14,12 +14,12 @@
 version="@PNGLIB_VERSION@"
 prefix="@prefix@"
 exec_prefix="@exec_prefix@"
-libdir="@libdir@"
+libdir=
 includedir="@includedir@/libpng@PNGLIB_MAJOR@@PNGLIB_MINOR@"
 libs="-lpng@PNGLIB_MAJOR@@PNGLIB_MINOR@"
 all_libs="-lpng@PNGLIB_MAJOR@@PNGLIB_MINOR@ @LIBS@"
 I_opts="-I${includedir}"
-L_opts="-L${libdir}"
+L_opts=
 R_opts=""
 cppflags=""
 ccopts=""
diff -Naur libpng-1.6.17/libpng.pc.in libpng-1.6.17.tpg/libpng.pc.in
--- libpng-1.6.17/libpng.pc.in	2015-03-26 13:06:23.000000000 +0000
+++ libpng-1.6.17.tpg/libpng.pc.in	2015-04-08 18:12:51.894356455 +0000
@@ -1,11 +1,11 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-libdir=@libdir@
+libdir=
 includedir=@includedir@/libpng@PNGLIB_MAJOR@@PNGLIB_MINOR@
 
 Name: libpng
 Description: Loads and saves PNG files
 Version: @PNGLIB_VERSION@
-Libs: -L${libdir} -lpng@PNGLIB_MAJOR@@PNGLIB_MINOR@
+Libs: -lpng@PNGLIB_MAJOR@@PNGLIB_MINOR@
 Libs.private: @LIBS@
 Cflags: -I${includedir}
diff -Naur libpng-1.6.17/scripts/libpng.pc.in libpng-1.6.17.tpg/scripts/libpng.pc.in
--- libpng-1.6.17/scripts/libpng.pc.in	2015-03-26 13:06:24.000000000 +0000
+++ libpng-1.6.17.tpg/scripts/libpng.pc.in	2015-04-08 18:13:25.309253393 +0000
@@ -1,10 +1,10 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-libdir=@libdir@
+libdir=
 includedir=@includedir@/libpng16
 
 Name: libpng
 Description: Loads and saves PNG files
 Version: 1.6.20
-Libs: -L${libdir} -lpng16
+Libs: -lpng16
 Cflags: -I${includedir}
