--- libpng-1.6.8/libpng-config.in.0003~	2013-12-19 16:04:50.000000000 +0100
+++ libpng-1.6.8/libpng-config.in	2014-01-14 19:37:08.683548804 +0100
@@ -14,12 +14,12 @@
 version="@PNGLIB_VERSION@"
 prefix="@prefix@"
 exec_prefix="@exec_prefix@"
-libdir="@libdir@"
+libdir=""
 includedir="@includedir@/libpng@PNGLIB_MAJOR@@PNGLIB_MINOR@"
 libs="-lpng@PNGLIB_MAJOR@@PNGLIB_MINOR@"
 all_libs="-lpng@PNGLIB_MAJOR@@PNGLIB_MINOR@ @LIBS@"
 I_opts="-I${includedir}"
-L_opts="-L${libdir}"
+L_opts=""
 R_opts=""
 cppflags=""
 ccopts=""
--- libpng-1.6.8/libpng.pc.in.0003~	2013-12-19 16:04:50.000000000 +0100
+++ libpng-1.6.8/libpng.pc.in	2014-01-14 19:37:08.683548804 +0100
@@ -1,11 +1,11 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-libdir=@libdir@
+libdir=
 includedir=@includedir@/libpng@PNGLIB_MAJOR@@PNGLIB_MINOR@
 
 Name: libpng
 Description: Loads and saves PNG files
 Version: @PNGLIB_VERSION@
-Libs: -L${libdir} -lpng@PNGLIB_MAJOR@@PNGLIB_MINOR@
+Libs: -lpng@PNGLIB_MAJOR@@PNGLIB_MINOR@
 Libs.private: @LIBS@
 Cflags: -I${includedir}
--- libpng-1.6.8/scripts/libpng.pc.in.0003~	2014-01-14 19:37:08.683548804 +0100
+++ libpng-1.6.8/scripts/libpng.pc.in	2014-01-14 19:37:27.760214866 +0100
@@ -1,10 +1,9 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-libdir=@libdir@
 includedir=@includedir@/libpng16
 
 Name: libpng
 Description: Loads and saves PNG files
 Version: 1.6.8
-Libs: -L${libdir} -lpng16
+Libs: -lpng16
 Cflags: -I${includedir}
