diff --git a/CMakeLists.txt b/CMakeLists.txt
index b825d90..6b9196f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -213,6 +213,7 @@ if(NOT WIN32 OR CYGWIN OR MINGW)
   set(libdir      ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR})
   set(includedir  ${CMAKE_INSTALL_PREFIX}/include)
   set(LIBS        "-lz -lm")
+  ADD_DEFINITIONS(-fPIC)
   configure_file(${CMAKE_CURRENT_SOURCE_DIR}/libpng.pc.in
     ${CMAKE_CURRENT_BINARY_DIR}/${PNGLIB_NAME}.pc @ONLY)
   CREATE_SYMLINK(${PNGLIB_NAME}.pc libpng.pc)
